packages/iTwinUI-react/src/core/Table/Table.tsx

```diff
@@ -178,6 +179,11 @@ export type TableProps<
    * Must be memoized.
    */
   onFilter?: (filters: TableFilterValue<T>[], state: TableState<T>) => void;
+  /**
+   * Value used for global filtering.
+   * Use with `globalFilter` and/or `manualGlobalFilter` to handle filtering yourself e.g. filter in server-side.
+   */
+  globalFilterValue?: any; // eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
Why does this need to be `any`?

Reply:
custom filter function can be used with any filtering value type as shown [here](https://react-table-v7.tanstack.com/docs/api/useGlobalFilter#table-options), so I didn't want to limit it to string only